music-journal/src/initFirebase.js

```diff
@@ -2,18 +2,21 @@ import firebase from 'firebase/app';
 import 'firebase/auth';
 
 const config = {
-apiKey: process.env.FIREBASE_API_KEY,
-authDomain: process.env.FIREBASE_AUTH_DOMAIN,
-databaseURL: process.env.FIREBASE_DATABASE_URL,
-projectId: process.env.FIREBASE_PROJECT_ID,
-storageBucket: process.env.FIREBASE_STORAGE_BUCKET,
-messagingSenderId: process.env.FIREBASE_MESSAGING_SENDER_ID
+    apiKey: process.env.FIREBASE_API_KEY,
```

Comment:
made my heart jump here, thought for a second you'd committed the credentials 😹

Reply:
Should just be formatting things rather than creds!